src/v2/Apps/Settings/Routes/EditSettings/SettingsEditSettingsInformation.tsx

```diff
@@ -170,70 +164,40 @@ export const UserInformation: React.FC<UserInformationProps> = ({
                   />
                 </>
               )}
-              {status && !status.success && (
+
+              {/* TODO: Remove with toast usage */}
+              {status && status.error && (
```

Comment:
The reason the red box was showing is that it was asserting on an attr that was never there to begin with. This is also a weird way to handle it in the first place. Will remove all of this in a refactor.